nextcord/guild.py

```diff
@@ -2083,6 +2084,28 @@ async def fetch_channel(self, channel_id: int, /) -> Union[GuildChannel, Thread]
         channel: GuildChannel = factory(guild=self, state=self._state, data=data)  # type: ignore
         return channel
 
+    async def fetch_onboarding(self) -> Onboarding:
```

Comment:
When a http get operation doesn't require a parameter (like an id), the method doesn't have `fetch_`. An example is `vanity_invite`.